servers/rendering/shader_language.cpp

```diff
@@ -394,6 +394,9 @@ const ShaderLanguage::KeyWord ShaderLanguage::keyword_list[] = {
 	{ TK_ERROR, nullptr, CF_UNSPECIFIED, {}, {} }
 };
 
+/**
+ * @brief Parse the next token from the code.
```

Comment:
We don't use these forms of documentation comments, should just be a normal comment or removed 🙂